indico/modules/events/registration/fields/simple.py

```diff
@@ -80,12 +81,25 @@ def view_data(self):
         items = deepcopy(self.form_item.versioned_data['choices'])
         for item in items:
             item['caption'] = self.form_item.data['captions'][item['id']]
-        return {'choices': items}
+        return {'choices': items, 'places_used': self.get_places_used()}
 
     @property
     def filter_choices(self):
         return self.form_item.data['captions']
 
+    @property
+    def validators(self):
+        def _check_number_of_places(form, field):
+            if field.data:
+                choices = self.form_item.versioned_data['choices']
+                captions = self.form_item.data['captions']
+                for k, v in field.data.iteritems():
+                    choice = next((x for x in choices if x['id'] == k))
```

Comment:
Add a `None` default. Getting a `StopIteration` here would likely result in hard to debug errors.